pom.xml

```diff
@@ -456,7 +456,7 @@
         </repository>
     </repositories>
     <properties>
-        <swagger-parser-version>1.0.9-SNAPSHOT</swagger-parser-version>
+        <swagger-parser-version>1.0.9</swagger-parser-version>
```

Comment:
Thanks for the PR. May I know why you need to use swagger parser 1.0.9 instead of 1.0.9-SNAPSHOT ?